src/EditorFeatures/Core/LanguageServer/Handlers/CodeActions/CodeActionHelpers.cs

```diff
@@ -154,7 +154,7 @@ internal static class CodeActionHelpers
 
             static LSP.Diagnostic[]? GetApplicableDiagnostics(LSP.CodeActionContext context, IUnifiedSuggestedAction action)
             {
-                if (action is UnifiedCodeFixSuggestedAction codeFixAction)
+                if (action is UnifiedCodeFixSuggestedAction codeFixAction && context.Diagnostics != null)
```

Comment:
real bug - if the client didn't pass us diagnostics with the code action request we would throw.  found during test cleanup